src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -81,26 +122,103 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
 
   const onEventBoxClick = useCallback(
     (e) => {
-      if (e.target.closest(`.${ScheduleEventBookmarkClass}`)) return;
+      if (
+        e.target.closest(`.${ScheduleEventBookmarkClass}`) ||
+        e.target.closest(`.${ScheduleEventExpandClass}`)
+      )
+        return;
 
       showEventModal();
     },
     [showEventModal]
   );
 
+  const handleMouseOver = useCallback(() => {
+    if (isMinimallyLongEvent) return;
+
+    showExpanded();
+  }, [isMinimallyLongEvent, showExpanded]);
+
+  const handleMouseOut = useCallback(() => {
+    if (isExpandedStatic) return;
+
+    hideExpanded();
+  }, [isExpandedStatic, hideExpanded]);
+
+  const handleCollapse = useCallback(() => {
+    hideExpandedStatic();
+    hideExpanded();
+  }, [hideExpanded, hideExpandedStatic]);
+
+  const handleToggleExpand = useCallback(() => {
+    if (!isExpandedStatic) {
+      toggleExpandedStatic();
+
+      return;
+    }
+
+    handleCollapse();
+  }, [isExpandedStatic, handleCollapse, toggleExpandedStatic]);
```

Comment:
What's the intent/logic here? Why is it using `toggleExpandedStatic` and `handleCollapse`?

Why are we calling `toggleExpandedStatic` only when `!isExpandedStatic` is `true`? Wouldn't we just call `showExpandedStatic` in that instance?

Reply:
probably this comment should clarify it a bit https://github.com/sparkletown/sparkle/pull/1733#discussion_r665959998
basically those are two different actions with different logic, thus it needs to work this way

Reply:
removed 2c9b3bd